docs/user/whats-new.asciidoc

```diff
@@ -24,4 +20,112 @@ coming[7.9.0]
+=== Enterprise Search comes to {kib}
+
+In 7.8, we reorganized the {kib} side navigation to make our Observability
+and Security apps easier to find. In 7.9, we added Enterprise Search
+to the side navigation. If you're an existing user of App Search and
+Workplace Search, you can now launch directly into those environments in {kib}.
+Check out https://www.elastic.co/guide/en/app-search/current/index.html[our documentation]
+for more information.
+
+[role="screenshot"]
+image::images/7.9-whats_new_search.png[Enterprise Search is side navigation]
+
+[float]
+[[lens-7-9]]
+=== Multiple y-axes in Lens
+
+Show two metrics on the y-axis at the same time in {kibana-ref}/lens.html[Lens].
+This enables you to compare and contrast data of vastly different types and ranges.
+For example, you might track the total page views for your web site,
+which tallies in the millions. You're also interested
+in the percentage of the web site bounce rates,
+typically between 1 and 100. You can stack both metrics on top of one
+another to quickly spot the outliers.
+
+Also in 7.9, Lens debuts color pick, sort, and a fitting function.
+
+[float]
+[[canvas-7-9]]
+=== Canvas variables for easy reuse
+
+You can now can create variables to easily reuse the
+strings and patterns in your Canvas elements. This is especially useful if you
+frequently copy and paste your elements. For example, when you clone a large workpad and need
+to connect your elements to a new index, you can use variables to quickly update the elements,
+instead of changing the index for each individual element. For instructions, refer to the
+{kibana-ref}/canvas.html[Canvas documentation].
+
+
+[float]
+[[dashboard-7-9]]
+=== Dashboard to Discover explorations
+
+Navigating between dashboards was a highlight in 7.8, and 7.9 adds
+the ability to navigate from a dashboard to Discover.  The time range,
+filters, and other parameters remain the same, so you can continue your
+analysis from a new perspective. For example,
+your dashboard might have a visualization for overall HTTP responses.
+You can navigate to Discover to see the saved search
+for HTTP responses. The new {kibana-ref}/explore-underlying-data.html[*Explore underlying data*]
+action is available for visualizations
+backed by a single index pattern.
+
+[role="screenshot"]
+image::images/explore_data_context_menu.png[Explore underlying data from panel context menu]
+
+[float]
+[[maps-7-9]]
+=== geo_shape fields at scale in Maps
+
+Maps introduces {kibana-ref}/maps-grid-aggregation.html[geo_shape fields with {es} aggregations].
+This means that you can plot your geo_shape fields at scale
+just like your geo_point fields.
+You can create a layer that clusters your lines and polygons until
+you zoom into an area with less than 10,000 discrete documents. Then,
+you’ll see individual lines and polygons.
+
+{kibana-ref}//tile-layer.html[Vector tile layers] are also enhanced&mdash;add layers to your map for data services that implement the
```

Comment:
Should this be a new section since vector tile layers are very different from geo_shape field clustering?